quasar/main.tf

```diff
@@ -208,3 +208,48 @@ resource "aws_security_group_rule" "rds-egress" {
   protocol          = "-1"                           # All protocols.
   cidr_blocks       = ["0.0.0.0/0"]
 }
+
+resource "aws_db_parameter_group" "quasar-qa" {
+  name   = "quasar"
+  family = "postgres10"
+
+  parameter {
+    name  = "rds_logical_replication"
+    value = "1"
+  }
+
+  parameter {
+    name  = "max_logical_replication_workers"
+    value = "500"
+  }
+}
+
+data "aws_ssm_parameter" "qa_username" {
+  name = "/quasar-qa/rds/username"
+}
+
+data "aws_ssm_parameter" "qa_password" {
+  name = "/quasar-qa/rds/password"
+}
+
+resource "aws_db_instance" "quasar-qa" {
+  allocated_storage     = 1000
+  storage_type          = "gp2"
```

Comment:
What's `gp2`? Would it be worth adding a comment here?

Reply:
It's default config, so removing this line.